packages/aws-cdk-lib/core/lib/nodejs-aspect.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable brace-style */
+import { Construct, IConstruct } from 'constructs';
+import { IAspect, Aspects } from './aspect';
+import * as cdk from '../../../aws-cdk-lib';
+import * as lambda from '../../aws-lambda';
+import { ReceiptRuleSet } from '../../aws-ses/lib/receipt-rule-set';
+import { Provider } from '../../custom-resources/lib';
+
+/**
+ * Runtime aspec
+ */
+abstract class RuntimeAspectsBase implements IAspect {
```

Comment:
This shouldn't live in `core`. How about putting all of this in `aws-lambda` since this aspect will act on Lambda functions?